src/model/comment.ml

```diff
@@ -5,18 +5,14 @@ module Identifier = Paths.Identifier
 type 'a with_location = 'a Location_.with_location
 
 
-type non_nest_aware_styles = [
+type style = [
   | `Bold
   | `Italic
+  | `Emphasis
```

Comment:
This appears as a partial reversion of #421 (see https://github.com/ocaml/odoc/pull/421/files#diff-02e8c64dc888acbac7e60f11e7ee5efd).

Reply:
Yes, it's on purpose, I reimplemented it in a different way in b6b59d1 . Imho, This is purely an HTML rendering issue, it should not affect the model or even the IR in any way. So now it doesn't.